playbooks/common/openshift-cluster/config.yml

```diff
@@ -22,6 +22,14 @@
   tags:
   - always
 
+- name: Set hostname
+  hosts: oo_masters_to_config:oo_nodes_to_config
+  tasks:
+  - name: Set hostname
+    hostname:
+      name: "{{ openshift.common.hostname }}"
+    when: openshift_set_hostname | default(True) | bool
```

Comment:
@sdodson This appears to be a regression from the previous behavior. Setting the hostname should not be required, with the exception of some certain edge cases. There were many user complaints with the previous default of always setting the hostname.